core/client/app/components/gh-search-input.js

```diff
@@ -135,60 +176,32 @@ export default Component.extend({
 
             if (selected.category === 'Posts' || selected.category === 'Pages') {
                 let id = selected.id.replace('post.', '');
-                transition = this.get('_routing.router').transitionTo('editor.edit', id);
+                this.get('_routing.router').transitionTo('editor.edit', id);
             }
 
             if (selected.category === 'Users') {
                 let id = selected.id.replace('user.', '');
-                transition = this.get('_routing.router').transitionTo('team.user', id);
+                this.get('_routing.router').transitionTo('team.user', id);
             }
 
             if (selected.category === 'Tags') {
                 let id = selected.id.replace('tag.', '');
-                transition = this.get('_routing.router').transitionTo('settings.tags.tag', id);
+                this.get('_routing.router').transitionTo('settings.tags.tag', id);
             }
-
-            transition.then(() => {
-                if (this.get('_selectize').$control_input.is(':focus')) {
-                    this._setKeymasterScope();
-                }
-            });
-        },
-
-        focusInput() {
-            this.get('_selectize').focus();
-        },
-
-        onInit() {
-            let selectize = this.get('_selectize');
-            let html = '<div class="dropdown-empty-message">Nothing found&hellip;</div>';
-
-            selectize.$empty_results_container = $(html);
-            selectize.$empty_results_container.hide();
-            selectize.$dropdown.append(selectize.$empty_results_container);
         },
 
         onFocus() {
             this._setKeymasterScope();
-            this.refreshContent();
         },
 
         onBlur() {
-            let selectize = this.get('_selectize');
-
             this._resetKeymasterScope();
-            selectize.$empty_results_container.hide();
         },
 
-        onType() {
-            let selectize = this.get('_selectize');
-
-            if (!selectize.hasOptions) {
-                selectize.open();
-                selectize.$empty_results_container.show();
-            } else {
-                selectize.$empty_results_container.hide();
-            }
+        search(term) {
+            return new RSVP.Promise((resolve, reject) => {
+                run.debounce(this, this._performSearch, term, resolve, reject, 600);
```

Comment:
After playing with it for a while I would suggest lowering the debounce period, `200ms` feels a lot more responsive even with a large dataset.